components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/par/OAuth2ParEndpoint.java

```diff
@@ -232,7 +234,11 @@ private void validateAuthzRequest(HttpServletRequest request) throws ParCoreExce
 
         try {
             OAuthAuthzRequest oAuthAuthzRequest = getOAuthAuthzRequest(request);
-            validateRequestObject(oAuthAuthzRequest);
+            RequestObject requestObject = validateRequestObject(oAuthAuthzRequest);
+            if (isFAPIConformantApp(oAuthAuthzRequest.getClientId())) {
+                EndpointUtil.validateFAPIResponseMode(oAuthAuthzRequest.getResponseType(),
```

Comment:
write a similar method to get the response type as well overriding the req obj parameters

Reply:
fixed with f22e2c736c5a4089d66f1f1e2c2c2c5cc6c1930a